iodata/test/test_wfn.py

```diff
@@ -137,8 +137,8 @@ def check_wfn(fn_wfn, nbasis, energy, charges_mulliken):
     if mol.mo.type == 'restricted':
         check_orthonormal(mol.mo.coeffs, olp, 1.e-5)
     elif mol.mo.type == 'unrestricted':
-        check_orthonormal(mol.mo.coeffs[:, :mol.mo.norb_a], olp, 1.e-5)
-        check_orthonormal(mol.mo.coeffs[:, mol.mo.norb_a:], olp, 1.e-5)
```

Comment:
I slightly prefer the old style (French grammar, adjective after the noun) because it follows the typical flow of object-oriented programming. With attribute access, one zooms in on specifics when going from left to right through an expression, e.g. `big_object.attribute.attribute_of_attribute.small_detail`. It always seemed logical to me to put main things first, followed by further specifications, but when one is more accustomed to English grammar, the old attribute names could be counterintuitive. I'm not sure what the best option is.

Reply:
@tovrstra I am not sure I get how this affects the code. Can you please show it in a snippet?